lib/game.rb

```diff
@@ -0,0 +1,56 @@
+class Game 
+  attr_reader :counter_frame, :user_input, :total_score_array, :frame_score
+  
+  def user_input_scores
+    puts "Please enter your scores for frame no.#{ @counter_frame } (need to between 0 and 10)"
+    @user_input = gets.chomp
+    raise "Score need to between 0 and 10" if @user_input.to_i > 10
+    return @user_input.to_i
+  end
+  
+  def frame_manager
+    @total_score_array = []
+    for i in 1..9 do
+      @counter_frame = i
+      roll_manager
+    end
+    tenth_frame
+  end
+
+  def tenth_frame
```

Comment:
Method names easy to understand, as you mentioned this morning, some refactoring would help make your methods shorter.